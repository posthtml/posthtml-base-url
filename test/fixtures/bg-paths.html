<head>
  <style>
    .test {
      background-image: url('image.jpg');
      background: url('image.jpg');
    }

    .test2 {
      background-image: url('https://preserve.me/image.jpg');
      background: url('https://preserve.me/image.jpg');
    }

    .multiple-bg {
      background-image: url('one.png'), url(two.png), url("three.png"), linear-gradient(to right, rgba(30, 75, 115, 1), rgba(255, 255, 255, 0));
    }
  </style>
</head>

<body style="background: url('image.jpg')">
  <table style="background: url(image.jpg)">
    <tr>
      <th style="background-image: url('image.jpg')">Header</th>
    </tr>
    <tr>
      <td style="background-image: url(image.jpg)">Cell</td>
    </tr>
  </table>
</body>
