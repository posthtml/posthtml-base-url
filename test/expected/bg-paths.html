<head>
  <style>.test {
      background-image: url('relative/image.jpg');
      background: url('relative/image.jpg');
    }

    .test2 {
      background-image: url('https://preserve.me/image.jpg');
      background: url('https://preserve.me/image.jpg');
    }

    .multiple-bg {
      background-image: url('relative/one.png'), url(relative/two.png), url("relative/three.png"), linear-gradient(to right, rgba(30, 75, 115, 1), rgba(255, 255, 255, 0));
    }</style>
</head>

<body style="background: url('relative/image.jpg')">
  <table style="background: url(relative/image.jpg)">
    <tr>
      <th style="background-image: url('relative/image.jpg')">Header</th>
    </tr>
    <tr>
      <td style="background-image: url(relative/image.jpg)">Cell</td>
    </tr>
  </table>
</body>
